src/file.rs

```diff
@@ -133,24 +135,23 @@ impl TryFrom<&[u8]> for Header {
 
 impl Header {
     /// 160 bytes should be added for Carbonado headers.
-    pub fn len() -> u64 {
+    pub fn len() -> usize {
```

Comment:
How do we decide to use usize in this case vs u64?
What is the reasoninig?

Reply:
Lengths are generally usize.